languages/emf-metamodel/src/test/resources/de/jplag/treeview/bookStore.ecore.treeview

```diff
@@ -1,13 +1,13 @@
-BookStorePackage : EPackage
-  BookStore : EClass
-    owner : EAttribute
-      EGenericType
-    location : EAttribute
-      EGenericType
-    books : EReference
-      EGenericType
-  Book : EClass
-    name : EAttribute
-      EGenericType
-    isbn : EAttribute
-      EGenericType
+EPackage #0: name="BookStorePackage", nsURI="http:///com.ibm.dyna...", nsPrefix="bookStore", eFactoryInstance=EFactory #0, eClassifiers=["BookStore", "Book"]
```

Comment:
I suspect this is due to EMF but its quite unmaintainable

Reply:
These are test resources, more specifically expected view files for certain test models. Thus, quite maintainable.